viewer/src/components/BoardAction.vue

```diff
@@ -1,61 +1,65 @@
 <template>
-  <g :class="['boardAction', kind, { highlighted, recent }]" v-b-tooltip :title="tooltip">
-    <SpecialAction
-      :class="{ faded }"
-      :planet="planet"
-      :action="boardActions[action].income"
-      :highlighted="highlighted"
-      :recent="recent"
-      :board="true"
-      x="-20"
-      y="-25"
-      width="40"
-      @click="onClick"
-    />
-    <g transform="translate(-15,-15)">
-      <image v-if="kind === 'power'" xlink:href='../assets/resources/power-charge.svg' width=20 :height=133/345*20
-      transform=" scale(-1,1) translate(-9, -12)" />
-      <rect
-        x="-8"
-        y="-8"
-        width="16"
-        height="16"
-        :rx="kind === 'power' ? 8 : 0"
-        :ry="kind === 'power' ? 8 : 0"
-        stroke="black"
-        stroke-width="1"
-        :fill="kind === 'power' ? '#984FF1' : 'green'"
-        transform="scale(0.8)"
-        v-if="costNumber > 1"
+  <svg width="80" height="80" viewBox="-28 -28 56 56" style="margin: -0.25rem; overflow: visible">
```

Comment:
The negative margin shouldn't be here - it should be decided by the parent component.

The svg should fit the width/height of the inside as best as possible. Then no negative margin is needed, and the parent component can just apply `mb-1` or `mr-1` classes to add margin, and be consistent with it: it can apply the same margin to `BoardAction` as with `SpecialAction`, ... 

Also instead of `width=80` and `height=80` - maybe use width / height that match the actions originially shown in the research board without the need to apply `scale`. This is the main place where `BoardActions` are displayed, `MoveButtons` are just secondary.